app/views/articles/_multiple_engagements.html.erb

```diff
@@ -6,5 +6,7 @@
     </span>
   <% end %>
 
-  <span class="placeholder" style="width: 24px; height: 24px;">&nbsp;</span>
+  <% if false %>
+  <%# <span class="placeholder" style="width: 24px; height: 24px;">&nbsp;</span> %>
+  <% end %>
```

Comment:
Oops I forgot to mention that you can take these out, since they will never be rendered